core/spec/models/spree/price_spec.rb

```diff
@@ -96,6 +96,33 @@
         expect(price.country).to eq(country)
       end
     end
+
+    describe '.currently_valid' do
```

Comment:
If I got it correctly, these are passing even without the change in the PR, isn't it?

Reply:
Right!!